### PR TITLE
Control api sidecar version improvement

### PR DESCRIPTION
**Pull request by whenhellfreezes**
**Please provide enough information so that others can review your pull request:**

1. Explain the **details** for making this change. What existing problem does the pull request solve? If it resolves an existing issue, be sure to use a Github [keyword](https://help.github.com/en/articles/closing-issues-using-keywords) to automatically close it.

The version of the sidecar for control api should not be determined by catalog.

<br/><br/>

2. What **changes to custom.yaml** are required?

NA

<br/><br/>

3. Have you documented any additional setup steps or configurations options?

NA

<br/><br/>

4. Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI.

Tested in Minikube.

<br/><br/>

### CODE REVIEW

File: gm-control-api/values.yaml [hunk 1]
```diff
@@ -108,7 +108,7 @@ gmControlApi:
     size:
 
 sidecar:
-  version: '{{- $.Values.global.catalog.sidecar.version | default "1.0.0" }}'
+  version: '{{- $.Values.global.gmControlApi.sidecar.version | default $.Values.global.sidecar.version }}'
```
Comment (sofiashalotenko) [inline gm-control-api/values.yaml]: What does that value actually refer to?  There doesn't seem to be a global.gmControlApi.sidecar.version in the file.

Comment (whenhellfreezes) [reply gm-control-api/values.yaml]: Yeah but the other charts do have those. So this makes it consistent and at worst defaults to something that is definitely there.

Comment (sofiashalotenko) [reply gm-control-api/values.yaml]: I guess my implied suggestion is maybe we add a gmControlApi.sidecar.version option in the yaml so it makes more sense--unless there's some reason not to.

Comment (whenhellfreezes) [reply gm-control-api/values.yaml]: So I added a change. The option is visible in greymatter.yaml but I think it is better to use the global version so I commented it out.

**Pull request closed without merging**